R/reportRenovation.R

```diff
@@ -0,0 +1,147 @@
+    reportAgg(v_renovation,
+              "Renovation|Residential|{typ} (bn m2/yr)", brickSets,
+              agg = c(bs = "all", hs = "all", bsr.hsr = "all", vin = "all", loc = "all", inc = "all"),
+              rprt = c(typ = "res"),
+              silent = silent),
+
+
+    ## by location ====
+    reportAgg(v_renovation,
+              "Renovation|Residential|{loc} (bn m2/yr)", brickSets,
+              agg = c(bs = "all", hs = "all", bsr.hsr = "all", vin = "all", typ = "res", inc = "all"),
+              rprt = c(loc = "all"),
+              silent = silent),
+
+    ## by final building shell ====
+    reportAgg(v_renovation,
+              "Renovation|Residential|{bsr} (bn m2/yr)", brickSets,
+              agg = c(bs = "all", hs = "all", hsr = "all0", vin = "all", loc = "all", typ = "res", inc = "all"),
+              rprt = c(bsr = "all"),
+              silent = silent),
+
+
+    ## by initial heating system ====
+    reportAgg(v_renovation,
+              "Renovation|Residential|Initial|{hs} (bn m2/yr)", brickSets,
+              agg = c(bs = "all", bsr = "all0", hsr = "all", vin = "all", loc = "all", typ = "res", inc = "all"),
+              rprt = c(hs = "all"),
+              silent = silent),
+
+
+    ## by building type + initial heating system ====
+    reportAgg(v_renovation,
+              "Renovation|Residential|{typ}|Initial|{hs} (bn m2/yr)", brickSets,
+              agg = c(bs = "all", bsr = "all0", hsr = "all", vin = "all", loc = "all", inc = "all"),
+              rprt = c(hs = "all", typ = "res"),
+              silent = silent),
+
+
+    ## by final heating system ====
+    reportAgg(v_renovation,
+              "Renovation|Residential|Final|{hsr} (bn m2/yr)", brickSets,
+              agg = c(bs = "all", hs = "all", bsr = "all0", vin = "all", loc = "all", typ = "res", inc = "all"),
+              rprt = c(hsr = "all"),
+              silent = silent),
+
+
+    ## by final heating system, without zero state ====
+    reportAgg(v_renovation,
+              "Renovation|Residential|Final with zero|{hsr} (bn m2/yr)", brickSets,
+              agg = c(bs = "all", hs = "all", bsr = "all0", vin = "all", loc = "all", typ = "res", inc = "all"),
+              rprt = c(hsr = "all0"),
+              silent = silent),
+
+    ## only identical replacement ====
+    reportAgg(v_renovation,
+              "Renovation|Residential|Identical replacement (bn m2/yr)", brickSets,
+              agg = c(bs = "all", hs.hsr = "identRepl", bsr = "all0", vin = "all", loc = "all",
+                      typ = "res", inc = "all"),
+              silent = silent),
+
+    ## only identical replacement by heating system ====
+    reportAgg(v_renovation,
+              "Renovation|Residential|Identical replacement|{hs.hsr} (bn m2/yr)", brickSets,
+              agg = c(bs = "all", bsr = "all0", vin = "all", loc = "all", typ = "res", inc = "all"),
+              rprt = c(hs.hsr = "identRepl"),
+              silent = silent)
+
```

Comment:
I added this, but found another issue: I don't know if you noticed that the variabel ```Renovation|Residential|Identical replacement (bn m2/yr)``` reports all renovations that are identical replacements of the heating system. While I find it useful to report this, the name is of course misleading. I changed the name to ```Renovation|Residential|Heating|Identical replacement (bn m2/yr)``` and added the corresponding shell variable ```Renovation|Residential|Shell|Identical replacement (bn m2/yr)```.

Reply:
Good point. I think that `Renovation|Residential|Shell|Identical replacement` is much les relevant. In fact, I would interpret these flows rather as maintenance renovation because you don't replace the entire building shell as you do it with the heating system. The naming is clearer with your suggestion in any case. We might want to think about renaming the shell variable in e future but it is okay for now.